spec/schema.json

```diff
@@ -486,7 +486,7 @@
         "is_on_register_in_country_formed_in": {
           "type": "boolean"
         },
-        "register_name": {
+        "public_register_name": {
```

Comment:
BeneficialOwnerGovernmentOrPublicAuthority does not have `public_register_name`, I think can be removed with `is_on_register_in_country_formed_in` and `registration_number`

Reply:
This is for ManagingOfficerCorporate

Reply:
Ops, you are right Mark! sorry mates

Reply:
We'll have to look at BeneficialOwnerGovernmentOrPublicAuthority separately in another PR, I can see that there is no register_name or public_register_name in the spec for it, but there is also no field for it currently in the web model, but there appears to be a field for it in the v5 prototype.

Reply:
Nope, the register fields are on `beneficial-owner-other`, `managing-officer-corporate` and `entity`

Reply:
Yes, you are right, I was looking at the corporate page! too confusing by half!